lib/cost_report.rb

```diff
@@ -24,23 +98,56 @@ def matching_clusters
     end
   end
 
-  def add_ht_item_to_freq_table(ht_item)
-    overlap = HtItemOverlap.new(ht_item)
-    overlap.matching_orgs.each do |organization|
-      @freq_table[organization.to_sym][overlap.matching_orgs.count] += 1
+  def total_hscore(member)
+    spm_total(member.to_sym) + mpm_total(member.to_sym) + ser_total(member.to_sym)
```

Comment:
Not sure I understand why "org" and "member" are being used when they are. If it's arbitrary, it's worth unifying, and if it's meaningful in some way, make it clear somewhere.

Reply:
Me neither. In theory, not every "org" is a "membe" so there is a distinction to be made, but in practice it's been pretty sloppy.

Reply:
For tasks like cost reporting and scrubbing it makes sense to talk about members, since that's something we only do for actual members. 
Then there are prospective members (who want a fee estimate). 
Then there are orgs that are not members that we may still have to at least account for, like keio, kennesaw, augusta.
Just talking out loud, not offering an actual clever solution.